### ISSUE TITLE
Add init container log/exec to pod details view

### ISSUE THREAD

**Issue by NOBLES5E** [headlamp-k8s/headlamp#1450]
We can see the init containers but cannot view their log and cannot open a shell.

**Comment by joaquimrocha** [comment]
Hmm, good point. We should list also the init containers in the container chooser of the log viewer/terminal.

**Comment by joaquimrocha** [comment]
Let's also list the ephemeral containers assuming that works.
We can use [nested lists](https://v4.mui.com/components/lists/#nested-list) in the container selector, to show the type of container. Like:
```
-------------------------
| Containers             |
|  My Container 1        |
|  My Container 2        |
| Init Containers        |
|  My Init 1             |
-------------------------
```

**Comment by joaquimrocha** [comment]
Fixed by #1517
